gravitee-apim-console-webui/src/components/alerts/alertEnginePromotion/alert-engine-promotion.html

```diff
@@ -0,0 +1,61 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="ae-promo">
+  <div>
+    <div>
+      <h2 class="promo-title">Get started with Alerts</h2>
+    </div>
+    <div class="image-content">
+      <img src="../../../assets/api-observability-dark.png" alt="promotion" class="promotion-image" />
+    </div>
+    <div class="info-content">
+      <div>
+        <label class="promo-label">Try Alert Engine</label>
+      </div>
+      <div>
+        <p class="text-paragraph">
+          Alert Engine allows you to isolate, understand, and remediate for API performance and security risks before they cause a problem
+          for your customers.
+        </p>
+
+        <span class="text-span">Contact us to get started with a trial of Gravitee’s Alert Engine.</span>
+      </div>
+      <div class="button-content">
+        <a
+          ng-href="{{$ctrl.learnUrl}}"
+          mat-icon-button
```

Comment:
I think that a `mat-stroked-button` will fit better the mock-ups

Reply:
fixed button and icon issue together